core/objects.go

```diff
@@ -325,6 +325,9 @@ type Challenge struct {
 	// Used by dvsni challenges
 	Validation *jose.JsonWebSignature `json:"validation,omitempty"`
 
+	// used by tls-sni-01
+	N int64
```

Comment:
You'll need to add a migration to the `sa/_db/migrations` to add a column for this.